src/containers/MapContainer/LeftLists/Filters/FeedFilters/FeedFilters.tsx

```diff
@@ -29,14 +54,26 @@ export function FeedFilters() {
           checked={eventChecked}
           Icon={Event}
           label={texts.content.map.filters.events}
-          onChange={onChange}
+          onChange={toggleEventsFilter}
           variant={variants.title2}
         />
       </S.SectionContainer>
       <S.Title>{texts.content.map.filters.actionTypes}</S.Title>
       <FeedActionTypesSectionFilters color={colors.main.primary} type={FilterEntourageType.ASK_FOR_HELP} />
       <Divider />
       <FeedActionTypesSectionFilters color={colors.main.blue} type={FilterEntourageType.CONTRIBUTION} />
+      <S.Title>{texts.content.map.filters.updatedBefore}</S.Title>
+      <S.CircleContainer>
+        {marks.map((mark) => (
+          <S.Circle
+            key={mark.value}
+            isActive={Boolean(selectedTimeRange === mark.value)}
```

Comment:
C'est déjà un boolean

```suggestion
            isActive={selectedTimeRange === mark.value}
```